### DISCUSSION TITLE
Status of project

### DISCUSSION THREAD

**Discussion by paulm17** [mui/pigment-css#249] [Discussions]
It would be nice to have some sort of blog post to give an update on the project.  Especially wrt the react conf video and any updates related to that!  Thanks

**Comment by ZakKa89** [comment]
https://github.com/mui/pigment-css/issues/388#issuecomment-2902532570

**Reply by paulm17** [reply]
This adds nothing to do the discussion.

**Reply by ZakKa89** [reply]
Disagree. You might not have seen that update so I'm just trying to help you out. You asked for an update on the project right? And I literally linked you to a more recent post where an update was given... It's a real clear update on the project I'd say.

**Reply by paulm17** [reply]
With just 30 seconds of your time.  You could have determined whether or not I had seen the thread.

I made a post on [on Mar 12](https://github.com/mui/pigment-css/issues/388#issuecomment-2717787800)

And you are replying to a post made almost last year.

So from my perspective, again you are adding nothing to the discussion.

**Reply by ZakKa89** [reply]
Let's agree to disagree. The post I linked is newer than your post, since it's from May 22 :) 

And I also found it useful not just for you, but for anybody stumbling upon your question in this discussion. 

I think you're being a dick for no reason.

**Reply by paulm17** [reply]
LMFAO.  Since I replied to the post.  I would have received all notifications.  I would have been made aware of the pause of the project.

Also, this is my thread.  If I say that it doesn't add any information, then it doesn't.  MMmmm kay?  😂 

> I think you're being a dick for no reason.

Now you are crying.
